lib/item.rb

```diff
@@ -1,3 +1,14 @@
 class Item
+attr_reader :name, :price
+
+    def initialize (info)
+        @name = info[:name]
+        @price = info[:price]
+        #takes out all weird characters and symbols then convert to float
+    end
+
+    def price
+        @price.gsub(/[^\d.]/, '').to_f
```

Comment:
Do you know what this `regex` is doing here! It works, but regex and can be confusing. Make sure you know whats going on with this code! You could have used the string method `.delete("$")` here to the same effect.